packages/l1-confirmation-checker/README.md

```diff
@@ -1,36 +1,41 @@
-# L1 Confirmation Checker Tutorial
+# Tutorial: Parent chain confirmation checker
 
-`l1 confirmation checker` is a simple demo of Arbitrum's transaction finality checker (used to check if transaction submitted to l1 or not).
+`l1-confirmation-checker` is a simple demo of Arbitrum's transaction finality checker (used to check if a transaction was already submitted to the parent chain or not).
```

Comment:
Since we change the title in main readme file, should we also change here so this may not make confusion?